packages/features/users/components/UserTable/BulkActions/DeleteBulkUsers.tsx

```diff
@@ -0,0 +1,50 @@
+import type { Table } from "@tanstack/react-table";
+import { BanIcon } from "lucide-react";
+
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import { Dialog, DialogTrigger, ConfirmationDialogContent, Button, showToast } from "@calcom/ui";
+
+import type { User } from "../UserListTable";
+
+interface Props {
+  table: Table<User>;
+}
+
+export function DeleteBulkUsers({ table }: Props) {
+  const { t } = useLocale();
+  const selectedRows = table.getSelectedRowModel().flatRows.map((row) => row.original); // Get selected rows from table
+  const utils = trpc.useContext();
+  const deleteMutation = trpc.viewer.organizations.bulkDeleteUsers.useMutation({
+    onSuccess: () => {
+      utils.viewer.organizations.listMembers.invalidate();
```

Comment:
Invalidate -> we may need to use optimistic UI when orgs get bigger. For now its not needed